DOCS/man/options.rst

```diff
@@ -2426,6 +2426,12 @@ Subtitles
         ``--sub-speed=25/23.976`` plays frame based subtitles which have been
         loaded assuming a framerate of 23.976 at 25 FPS.
 
+``--sub-detect-rtl=<yes|no>``
+    Default: no. Set the ``Encoding`` flag to ``-1`` to enable Fribidi's base
```

Comment:
considering it only applies to converted subs (and thus VSFilter is not relevant), in which situation would you possibly want to set this to `no`?

Reply:
I do not know how well Fribidi's auto detection algorithm works, so I'm scared of introducing a change in default that might possibly regress some LTR subtitles. If you think it should be enabled by default then I could change that. also cc @avih 

I should update the description, however.

Reply:
I was thinking that either:
* it works correctly, then it can be enabled and we don't need an option
* it doesn't work correctly, then the entire PR isn't really that useful

That it regresses any LTR content is very unlikely, Unicode has clear definitions which glyphs/scripts are RTL. You'd have to mess up really bad.